### DISCUSSION TITLE
Your best palettes - V1.0.0 - V1.0.2

### DISCUSSION THREAD

**Discussion by Loupimo** [Loupimo/UniversalSwatchSlots#8] [Ideas]
Hello there !
In this discussion I invite you to share your JSON configurations containing the palettes of your own in order to give some nice ideas to other people 😄 

Don't hesitate to post screenshots to illustrate the beauty of your palettes.
Be creative !

Note: I will try my best to keep the same file structure throughout updates. Current supported format: from 1.0.0 to 1.0.2

**Comment by CTRQuko** [comment]
I have created a file from a list of colors that I already used, [https://www.reddit.com/r/SatisfactoryGame/comments/154vft6/vencams_colour_list_25/](url), I have not been able to contact the author, but as it is a public list I don't think there is a problem.
[VELCAM COLOUR LIST.json](https://github.com/user-attachments/files/18137792/VELCAM.COLOUR.LIST.json)
Fixed!
